openshift/4.0/pipelines/Jenkinsfile.cicd

```diff
@@ -174,57 +174,57 @@ pipeline {
         }
       }
     }
-			
-
-    stage('Tests') {
-      when {
-        expression { new Boolean(env.SKIP_BUILD) == false }
-      }
-      options { timeout(time: 20, unit: 'MINUTES') }
-      failFast true
-      parallel {
-        stage('Test Frontend') {
-          agent { label 'jenkins-slave-npm' }
-          steps {
-            script {
-              // pull code
-              checkout scm
 
-              echo 'Preparing the report for the publishing ...'
-              unstash name: 'zap'
 
-              commonPipeline.runFrontendTests()
-
-              dir(DEVOPS_DIRECTORY) {
-                sh "ZAP_REPORT=${WORKSPACE}/zap-output/zap-report.xml HTML_ZAP_REPORT=${WORKSPACE}/zap-output/zap-report.html ./player.sh scan ${OC_JOB_NAME} -apply"
-              }
-              echo 'Frontend lint checks and tests passed'
-            }
-          }
-          post {
-            failure {
-              error '*** Frontend lint checks and tests failed ***'
-            }
-          }
-        }
-        stage('Test Backend') {
```

Comment:
unless I'm mistaken, this is the only part that needs to be disabled, the frontend tests should be fine.

Reply:
I've removed both as we should be able to use the GitHub Action to do the SonarQube scan.  This will speed up the pipeline and we already do the tests in GitHub.